app.py

```diff
@@ -242,7 +244,6 @@ def cli(
     only_dependency_monkey_documents,
     only_adviser_documents,
     only_provenance_checker_documents,
-    metrics_pushgateway_url,
```

Comment:
Please remove `--metrics-pushgateway-url` from CLI options (CLI will fail otherwise as `metrics_pushgateway_url` argument will be passed here by Click).